examples/tutorials/vectors.py

```diff
@@ -0,0 +1,377 @@
+"""
+Plot vectors
+==========
+
+Plotting vectors is handled by :meth:`pygmt.Figure.plot`.
+
+.. note::
+
+    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+    To see the figures while using a Python script instead, use
+    ``fig.show(method="external")`` to display the figure in the default PDF viewer.
+
+    To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
+    is the desired name and file extension for the saved figure.
+"""
+# TODO: change this number to reflect the correct thumbnail
+# sphinx_gallery_thumbnail_number = 3
+
+import numpy as np
+import pygmt
+
+########################################################################################
+# Plot Caretesian Vectors
+# ----------
+#
+# Create a simple Cartesian vector using a starting point through
+# ``x``, ``y``, and ``direction`` parameters. The direction is specified
+# by a list of two 1d arrays structured as ``[[angle_in_degrees], [length]]``
+#
+# On the shown figure, the plot is projected on a _10cm X 10cm_ region,
+# which is specified by the `region` and `projection` parameters.
+# The magnitude of the vector also uses centimeters by default but
+# could be changed using [pygmt.config](https://www.pygmt.org/latest/api/generated/pygmt.config.html#pygmt-config)
```

Comment:
```suggestion
# could be changed using :meth:`pygmt.config`
```
Just a quick note that you can use the referencing styles listed in the [cross-referencing with Sphinx contributing section](https://github.com/GenericMappingTools/pygmt/blob/master/CONTRIBUTING.md#cross-referencing-with-sphinx) for internal references.

Reply:
Thanks for pointing that out! I just added it to the file